NDB.Covid19/NDB.Covid19.Droid/Views/Settings/ConsentSettingPageFragment.cs

```diff
@@ -12,60 +12,58 @@ public class ConsentSettingPageFragment : Fragment
-            TextView consentTwoTitle = RelativeLayout2.FindViewById<TextView>(Resource.Id.consent_page_title);
-            TextView consentThreeTitle = RelativeLayout3.FindViewById<TextView>(Resource.Id.consent_page_title);
             TextView consentFourTitle = RelativeLayout4.FindViewById<TextView>(Resource.Id.consent_page_title);
-            TextView consentFiveTitle = RelativeLayout5.FindViewById<TextView>(Resource.Id.consent_page_title);
-            TextView consentSixTitle = RelativeLayout6.FindViewById<TextView>(Resource.Id.consent_page_title);
-            TextView consentSevenTitle = RelativeLayout7.FindViewById<TextView>(Resource.Id.consent_page_title);
-            TextView consentEightTitle = RelativeLayout8.FindViewById<TextView>(Resource.Id.consent_page_title);
-            TextView consentNineTitle = RelativeLayout9.FindViewById<TextView>(Resource.Id.consent_page_title);
+            RelativeLayout4.FindViewById<TextView>(Resource.Id.consent_page_text).TextFormatted = HtmlCompat.FromHtml(ConsentViewModel.CONSENT_FOUR_PARAGRAPH, HtmlCompat.FromHtmlModeLegacy);
 
-            consentOneTitle.Text = ConsentViewModel.CONSENT_ONE_TITLE;
-            consentTwoTitle.Text = ConsentViewModel.CONSENT_TWO_TITLE;
-            consentThreeTitle.Text = ConsentViewModel.CONSENT_THREE_TITLE;
-            consentFourTitle.Text = ConsentViewModel.CONSENT_FOUR_TITLE;
-            consentFiveTitle.Text = ConsentViewModel.CONSENT_FIVE_TITLE;
-            consentSixTitle.Text = ConsentViewModel.CONSENT_SIX_TITLE;
-            consentSevenTitle.Text = ConsentViewModel.CONSENT_SEVEN_TITLE;
-            consentEightTitle.Text = ConsentViewModel.CONSENT_EIGHT_TITLE;
-            consentNineTitle.Text = ConsentViewModel.CONSENT_NINE_TITLE;
+            //ABOUT
+            _aboutHeader = view.FindViewById<TextView>(Resource.Id.consent1_about_header);
+            _aboutHeader.Text = ConsentViewModel.CONSENT_ONE_TITLE;
+            _aboutText1 = view.FindViewById<TextView>(Resource.Id.consent1_about_text_section1);
+            _aboutText1.Text = ConsentViewModel.CONSENT_ONE_PARAGRAPH_SECTION_ONE;
+            _aboutText2 = view.FindViewById<TextView>(Resource.Id.consent1_about_text_section2);
+            _aboutText2.Text = ConsentViewModel.CONSENT_ONE_PARAGRAPH_SECTION_TWO;
 
-            consentOneTitle.ContentDescription = ConsentViewModel.CONSENT_ONE_TITLE.ToLower();
-            consentTwoTitle.ContentDescription = ConsentViewModel.CONSENT_TWO_TITLE.ToLower();
-            consentThreeTitle.ContentDescription = ConsentViewModel.CONSENT_THREE_TITLE.ToLower();
-            consentFourTitle.ContentDescription = ConsentViewModel.CONSENT_FOUR_TITLE.ToLower();
-            consentFiveTitle.ContentDescription = ConsentViewModel.CONSENT_FIVE_TITLE.ToLower();
-            consentSixTitle.ContentDescription = ConsentViewModel.CONSENT_SIX_TITLE.ToLower();
```

Comment:
do not remove content description as it is required for proper accessibility

Reply:
@nc-kano resolved